src/routes/members.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const membersController = require('../controllers/members');
+const validations = require('../validations/members');
+
+const router = express.Router();
+
+router
+  .get('/', membersController.getAllMembers)
+  .post('/', validations.validateMemberCreation, membersController.createMember)
+  .put('/:id', validations.validateMemberCreation, membersController.updateMember);
```

Comment:
I'm going to switch to update validation in the merge (this is because my teammate has that validation).